docs/user_guide/platform_dependence.rst

```diff
@@ -7,7 +7,7 @@ Per-Platform Target Configuration
 
 The following lists and sections of the target configuration can be specified individually per platform:
 
-- `include_directories` and `public_include_directories`
+- `include_directories` and `include_directories_public`
```

Comment:
In the issue you described it the other way around

Reply:
This PR corrects the documentation, the key is currently still called `include_directories_public`. This needs to be changed in order to complete issue 113.